Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalMinBias_Output_cff.py

```diff
@@ -10,10 +10,9 @@
     ),
     outputCommands = cms.untracked.vstring(
 	'keep *_gtDigisAlCaMB_*_*',
-        'keep HBHERecHitsSorted_hbherecoMB_*_*',
-        'keep HORecHitsSorted_horecoMB_*_*',
-        'keep HFRecHitsSorted_hfrecoMB_*_*',
-        'keep HFRecHitsSorted_hfrecoMBspecial_*_*',
```

Comment:
"special" is no longer used. Cant you see these lines belong to the older version - not the new one